tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -201,6 +201,41 @@ def test_fn_to_docker_container(self, ondemand_aws_cluster):
     ####################################################################################################
     # Status tests
     ####################################################################################################
+
+    @pytest.mark.level("minimal")
+    @pytest.mark.clustertest
+    def test_status_scheduler_basic_flow(self, cluster):
+
+        cluster.save()
+        # the scheduler start running in a delay of 1 min, so the cluster startup will finish properly.
+        # Therefore, the test needs to sleep for a while.
+        time.sleep(60)
```

Comment:
is there an easy way to adjust the time of the scheduler to much shorter, rather than having to have this test wait over a minute to run?

Reply:
yeah, I'll remove the sleeping time, cause the scheduler does not start running in delay any more. ([1131](https://github.com/run-house/runhouse/pull/1131/files)